core/dbt/tests/fixtures/project.py

```diff
@@ -120,6 +120,11 @@ def dbt_profile_target():
     }
 
 
+@pytest.fixture(scope="class")
+def profile_user(dbt_profile_target):
```

Comment:
Which adapter need this change and why?

Reply:
We need a way to get the "owner" in the catalog. It's different for different adapters. Some of them use the "user" in the profile, some use other things. So this is special fixture to help with that. We didn't have a standard way to get the profile's user before.